### PR TITLE
Update ingress.yaml

### PR DESCRIPTION
**Pull request by melakonda**
Ingress servicePort is coming as http. I changed it to {{ .Values.service.port }}

**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/grafana/templates/ingress.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
             {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
             pathType: {{ .Values.ingress.pathType }}
             {{- end }}
-            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "http" "context" $)  | nindent 14 }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "{{ .Values.service.port }}" "context" $)  | nindent 14 }}
```
Comment (javsalgar) [inline bitnami/grafana/templates/ingress.yaml:32]: This change is not necessary. Using the named port should be enough. Could you explain the issue you are having?

**Review by javsalgar — CHANGES_REQUESTED** [review]

File: bitnami/grafana/templates/ingress.yaml (hunk 1 above)
Comment (1993Saurabh) [reply bitnami/grafana/templates/ingress.yaml:32]: Hi @javsalgar ,

Could you please explain why using the port named is enough in the servicePort. Due to this issue the port number in the ingress file is coming like this and sadly I am unable to open the rabbitmq management console:

image

But once I manually fixed this issue in my local machine and redeployed the ingress, it got changed and was able to open the management console after that:

image


So could you please explain why this is not an issue in the Chart itself and there is no way to override this value.

Comment (javsalgar) [reply bitnami/grafana/templates/ingress.yaml:32]: When setting an ingress rule, the named port is enough for Ingress for recognizing the service port. I just deployed it with ingress.enabled=true and could access without problems.

image

Comment (1993Saurabh) [reply bitnami/grafana/templates/ingress.yaml:32]: Hi @javsalgar ,

Sadly this setting is not working with the Azure Application Gateway, you probably tested it with the nginx-controller. If it is then can you please help how can I do the same with Azure App Gateway? 

P.S: Azure App Gateway does not allow text in the Port.

Comment (javsalgar) [reply bitnami/grafana/templates/ingress.yaml:32]: Hi,

If that's the case, we should update all of our Ingress rules. Do you have a link where they confirm that incompatibility with named ports? I'll need it to forward it to the rest of the engineering team for evaluation.

Comment (1993Saurabh) [reply bitnami/grafana/templates/ingress.yaml:32]: See if this link can help : [Link](https://github.com/Azure/application-gateway-kubernetes-ingress/blob/master/docs/annotations.md)

But we definitely need the updation as Azure Application Gateway is right away discarding the named port at the moment.

Comment (javsalgar) [reply bitnami/grafana/templates/ingress.yaml:32]: I opened a ticket so they can confirm https://github.com/Azure/application-gateway-kubernetes-ingress/issues/1139

Comment (1993Saurabh) [reply bitnami/grafana/templates/ingress.yaml:32]: okay thanks

Comment (javsalgar) [reply bitnami/grafana/templates/ingress.yaml:32]: Still no answer. Let's give it some extra days before continuing with this.

Comment (javsalgar) [reply bitnami/grafana/templates/ingress.yaml:32]: I pinged again

Comment (1993Saurabh) [reply bitnami/grafana/templates/ingress.yaml:32]: Yeah, and I opened the same question [here](https://stackoverflow.com/questions/66204759/does-azure-application-gateway-ingress-controller-supports-named-ports) as well

Comment (javsalgar) [reply bitnami/grafana/templates/ingress.yaml:32]: Let's do something to not block you. For the time being, I will accept this change in the chart, but I won't change it in the rest until we have a confirmation from Azure about this. Could you bump the minor version of the chart?

Comment (1993Saurabh) [reply bitnami/grafana/templates/ingress.yaml:32]: @javsalgar could you please tell me the version of the chart which you bumped? We need to freeze the version in our project.

Comment (javsalgar) [reply bitnami/grafana/templates/ingress.yaml:32]: Hi! We didn't bump the version, could you do it in this PR? The minor version would be required.

**Pull request closed without merging**